using-timescaledb/data-retention.md

```diff
@@ -20,6 +26,68 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
```

Comment:
```suggestion
### Data Retention on Continuous Aggregates
```

Reply:
Not entirely accurate: this is also about data retention for the raw table that the continuous aggregate is using.

Reply:
I am confused. With raw table, do you mean the "internal" hypertable which provides storage for the view or the non-aggregated table which you define the aggregate on (i.e., `conditions`)? I assume the latter, and in that case I don't think this is about retention on that non-aggregated table because dropping chunks on the aggregate shouldn't affect the non-aggregated table.

This relates to the confusion about `cascade_to_materialization`

Reply:
No, but dropping chunks on the non-aggregated table affects the aggregated table (or can, at least, if you set it up correctly), so the section does not only cover data retention on continuous aggregates.

Reply:
try 'with' then.

Reply:
Done.